### PR TITLE
Bug 1896302: Fix 4.6 test failures

### PR DESCRIPTION
**Pull request by gouyang**
This will fix/improve below test failures:
1. fix migration tests as it require rootdisk's accessMode is RXW.
2. fix cloud-init test as the wizard procedure to custom script instead of form.
3. increase vm stop timeout to 220 which is larger than vm's termination grace period 210s.
4. add checking to look for VM login prompt when starts a VM.
5. change container image from 'fedora-cloud-registry-disk-demo' to 'fedora-cloud-container-disk-demo'
6. fix network dialog test failures due to new UI changes.
7. add test ID for guest data test.
8. move boot-order test to bottom in next-run-configuration test as it has side effects.
9. add a check for loading vm disk page.

TODO: add test ID for test `vm.next.run.configuration.scenario.ts`.

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts:84]: I am getting this

```
 Error: Command failed: expect ./packages/kubevirt-plugin/integration-tests/tests/utils/scripts/expect-login.sh vm-list-actions-test-ayeaf test-ayeaf
   couldn't read file "./packages/kubevirt-plugin/integration-tests/tests/utils/scripts/expect-login.sh": no such file or directory
```

we probably need a better path resolution

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts:84]: also the expect-login script was not meant for checking(expecting) anything originally. Only for waiting for successful boot. 

That means it will not fail if the VM does not boot succesfully. We need return non null return code by the script (eg https://stackoverflow.com/a/58222144)

**Review by suomiy — CHANGES_REQUESTED** [review]

File: frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts
Comment (suomiy) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts:84]: also are we planning to add expect-login to other actions as well? like restart/migrate/ etc. ?

File: frontend/packages/kubevirt-plugin/integration-tests/views/wizard.view.ts
Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests/views/wizard.view.ts]: move it to `uiResource.view.ts`

File: frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts
Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts]: Fixed expect-login script and move it into `waitForStatus`, so other actions which has start action will have the check as well.

**Comment by gouyang** [comment]
More fixes:
1. test ID added for test `vm.next.run.configuration.scenario.ts`.
2. close wizard in `wizard` last test so it not affect other test.
3. adjust cdi test as it changes again.
4. disable to start a PXE vm.

@suomiy this is ready for review, could you take a look at it once again.

File: frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts:190]: it might be easier to just call waitForStatus here now.

File: frontend/packages/kubevirt-plugin/integration-tests/views/wizard.view.ts
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/views/wizard.view.ts:89]: we could move these table and kebab selectors as well

File: frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts
Comment (suomiy) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/models/baseVirtualMachine.ts]: I fixed the path on my machine, since I had it misconfigured.
But nevertherless, it is quite hardcoded for my taste (would be better to resolve the working directory on runtime). We can let it be for now...

File: frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts
Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts:190]: Done.

File: frontend/packages/kubevirt-plugin/integration-tests/views/wizard.view.ts
Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests/views/wizard.view.ts:89]: There are very similar files like `uiResource.view` and `kubevirtUIResource.view`, I'd like to go through them in a new PR,
as this one is focusing on fixing failures.

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/integration-tests/views/wizard.view.ts:89]: ok

File: frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts
Comment (suomiy) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts:190]: can you push the latest changes with this fix?

Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts]: ah, I forgot to do it, pushed the latest changes by now.

**Comment by suomiy** [comment]
/hold

until https://github.com/openshift/console/pull/7221 is merged

**Comment by gouyang** [comment]
@suomiy I removed the `expect` part in vm actions, so it can pass the tests for the moment.
Will add them back when 'expect' is ready.

**Comment by suomiy** [comment]
@gouyang ok, can you please at least comment out the places where expect would be used instead of deleting them? And adding a TODO marker so we can easily return to this later?

fyi, the expect PRs https://github.com/openshift/release/pull/13715 and https://github.com/openshift/console/pull/7306 looked promising, but it seems analyze script is broken and need to be fixed before this can get merged

**Comment by gouyang** [comment]
> @gouyang ok, can you please at least comment out the places where expect would be used instead of deleting them? And adding a TODO marker so we can easily return to this later?
> 
Comment it out instead of removing it and adding a TODO there.

**Pull request merged**
